### PR TITLE
Document annotation pv.kubernetes.io/bound-by-controller

### PR DESCRIPTION
**Pull request by afzal442**
Signed-off-by: afzal442 <afzal442@gmail.com>

Fixes: #36191

### CODE REVIEW

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 1]
```diff
@@ -437,6 +437,19 @@ Used on: PersistentVolumeClaim
...
 This annotation will be added to dynamic provisioning required PVC.
 
+### pv.kubernetes.io/bound-by-controller
+
+Example: `pv.kubernetes.io/bound-by-controller: "true"`
+
+Used on: PersistentVolume or PersistentVolumeClaim
+
+This annotation applies to {{< glossary_tooltip text="PersistentVolumes" term_id="persistent-volume" >}}
+(PVs) and {{< glossary_tooltip text="PersistentVolumeClaims" id="persistent-volume-claim" >}} (PVCs).
```
Comment (sftim) [inline content/en/docs/reference/labels-annotations-taints/_index.md:447]:
```suggestion
(PVs) and {{< glossary_tooltip text="PersistentVolumeClaims" term_id="persistent-volume-claim" >}} (PVCs).
```

**Review by sftim — COMMENTED** [review]
This change isn't valid. Try this fix?

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 2]
```diff
@@ -437,6 +437,19 @@ Used on: PersistentVolumeClaim
...
+When this annotation is present, that indicates that thWhee binding (either PV→PVC or PVC→PV) was installed
```
Comment (Kartik494) [inline content/en/docs/reference/labels-annotations-taints/_index.md:448]: Hi @afzal442 i am curious on the term `thWhee binding` could you let me know  whether it  is a typo or reference to it.

File: content/en/docs/reference/labels-annotations-taints/_index.md (hunk 1 above)
Comment (afzal442) [reply content/en/docs/reference/labels-annotations-taints/_index.md:447]: Sure! Thanks Tim! All done.

File: content/en/docs/reference/labels-annotations-taints/_index.md (hunk 2 above)
Comment (afzal442) [reply content/en/docs/reference/labels-annotations-taints/_index.md:448]: good catch @Kartik494! now look!

**Comment by sftim** [comment]
I'd like confirmation from SIG Storage that this PR is technically accurate.

File: content/en/docs/reference/labels-annotations-taints/_index.md (hunk 2 above)
Comment (Kartik494) [reply content/en/docs/reference/labels-annotations-taints/_index.md]: It's look fine from the previous changes.

**Comment by afzal442** [comment]
Sure @sftim! LMK when you get it approved.

**Comment by jsafrane** [comment]
lgtm from sig-storage POV.

**Comment by sftim** [comment]
/retitle Document annotation pv.kubernetes.io/bound-by-controller

**Comment by sftim** [comment]
Ah, actually, it's already landed in another PR: https://github.com/kubernetes/website/pull/37928

We should close this.

/close

**Pull request closed without merging**